### ISSUE TITLE
Syncing gets stuck on "mismatch at height 1000"

### ISSUE THREAD

**Issue by cilphex** [lightninglabs/lightning-app#646]
### Description

Lightning app never finishes syncing.

### Steps to reproduce the behavior

Install lightning app and open it.

### Expected behavior

Syncing should complete.

### Actual behavior

Syncing never completes. Logs show something like this on repeat:

```
2018-09-10 11:25:02.918 [WRN] BTCN: Detected mismatch at index=0 for checkpoints!!!
lnd: 2018-09-10 11:25:03.043 [WRN] BTCN: mismatch at height 1000, expected 8a116495a22639806b55c7fad5f9c6cd031cb41005e52a4afd516b5229e033ff got 20fdcc13d0563fac7b86e070d5e818a5f68974e4f35855e5937b1ca04e3b3522
```

### Screenshot and/or logs if applicable

![screen shot 2018-09-10 at 11 14 14 am](https://user-images.githubusercontent.com/611803/45316460-fd9c6700-b4eb-11e8-8eb7-c424714190aa.png)

Gist:

https://gist.github.com/cilphex/5394932ff27c797659fd6af7bb91e8a6

Version:

![screen shot 2018-09-11 at 2 35 24 pm](https://user-images.githubusercontent.com/611803/45389158-01a1b500-b5d0-11e8-8405-450a150850f0.png)

**Comment by Roasbeef** [comment]
Are you running a node locally?

**Comment by cilphex** [comment]
Not running a separate node locally, no.

**Comment by tanx** [comment]
Not entirely sure if it's the same issue but I got stuck too on the wait spinner before the fund wallet screen. Seems like the `NewAddress` grpc call never finished. Maybe another deadlock in lnd?

**Comment by Roasbeef** [comment]
Does restarting the app resolve this? As a temporary work around, you may need to clear the app state on disk on your machine. We have a set of bug fixes in neutrino that are staged hopefully for the next release of the application.

**Comment by cilphex** [comment]
Restarting doesn't help, unfortunately. Someone pointed me to the app state location for OSX but it wasn't accurate, so not sure where that's located. Happy to hear a fix is in the works! Should the fix work for me if I run the new version, or will I still need to clear the app state after switching?

**Comment by cilphex** [comment]
I saw that the ticket is still on the projects board, so I didn't try. I just removed the app directory and it started from scratch, working.

**Comment by tanx** [comment]
Should be fixed in `v0.2.4-alpha`
